contrib/diurnal/Makefile

```diff
@@ -0,0 +1,24 @@
+.PHONY: build push vet test clean
+
+TAG = 0.4.2
```

Comment:
minor: it's simpler to keep tags to 1 or 2 digits unless you're seriously doing development on this.  v0.4, v0.5...v1.0